docs/toc.md

```diff
@@ -24,40 +50,19 @@
 ## [🔧 Reflection & code generation](languages/csharp/reflection.md)
```

Comment:
Line 47 - should we remove the wrench from that one? Looks like asynchronous programming already has content